### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by yichuandoris** [yichuandoris/MI-449-SS17-740-js-firebase-intro-T4LtxO#1]
@egillespie Can you take a look at this? It's [hosted here](https://wangyi70-kgucykegmdzxwt4ltxo.surge.sh/) and meets the following criteria:

- [x] Use [this code](https://gist.github.com/egillespie/942cc3aed19977590b431c10333736c2) as a foundation for your project
- [x] Your own Firebase project is used to store all of the site's messages
- [x] A message and its creation time are added to the database when Enter is pressed or Woof is clicked
- [x] Messages are stored in the database according to the structure in `model.json`
- [x] Anyone can edit any message by clicking the pencil icon, typing a new message, and pressing Enter
- [x] Anyone can delete any message by clicking the red X icon
- [x] When a message is added, edited, or deleted, _Recent woofs_ automatically updates for everyone viewing the page
- [x] Use the `TODO` comments in `woofer-db.js` as your guide for placing code

Thanks for your help and I am a bit uncertain about the last line -->
function deleteWoofFromDatabase (woofKey) {
firebase.database().ref('woofs').child(woofKey).remove()
}
I originally wrote like this:
function deleteWoofFromDatabase (woofKey) {
firebase.database().ref('woofs').child(woofKey).remove()
}
but it turned out people can delete everything once they click on the "X" button, so I revised it.
Just want to make sure I am right with my latest version. Thanks! - Doris

**Comment by egillespie** [comment]
Yeah, `remove` can be pretty dangerous! It will remove the entire reference so if you aren't specific enough, say goodbye to your whole database! 😱😱😱😱😱

Good observation! :) The rest of your code looks great, too. :shipit:
